Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM continuumio/miniconda3:4.7.12
+FROM condaforge/mambaforge:latest
 
```

Comment:
Might be better to pin to [`condaforge/mambaforge:4.10.1-0`](https://hub.docker.com/r/condaforge/mambaforge/tags?page=1&ordering=last_updated)...